redis/asyncio/cluster.py

```diff
@@ -1206,7 +1211,7 @@ async def close(self, attr: str = "nodes_cache") -> None:
         )
 
 
-class ClusterPipeline(AbstractRedis, AbstractRedisCluster, AsyncRedisClusterCommands):
+class ClusterPipeline(RedisCluster):
```

Comment:
Why are the bases for this class changed? Inheriting directly from the RedisCluster class could lead to collisions in the namespace